package.json

```diff
@@ -21,12 +23,23 @@
   "bugs": "https://github.com/thepassle/custom-elements-ssr",
   "homepage": "",
   "scripts": {
+    "build": "pnpm terser dsd-polyfill.js -o dsd-polyfill.min.js",
```

Comment:
would you mind reverting this to npm?

Reply:
Astro and vite users are commonly very `pnpm` affine and i personally also do not use anything else anymore.

But concerning this particular operation.

First i found out, that `vite` resp. `esbuild` generates in this particular significant smaller minified code than `terser` and later I stumbled over a few issues related to the polyfill handling.

At the end i wrote a tiny vite-plugin for astro to inject the polyfill during the build process or while serving in dev mode directly as a generated asset. This seems to work much more reliable. You'll find it in another branch: https://github.com/mash-graz/custom-elements-ssr/blob/happydom/src/inject-dsd-polyfill.ts